fintech-examples/fintech-ui/src/app/bank/settings/settings.component.html

```diff
@@ -36,6 +45,15 @@ <h4 class="text-secondary">List of Transactions</h4>
             <input type="radio" value="{{ lotFromTppWithNewConsent }}" formControlName="lot" id="lot2" />
             <label for="lot2">Discard propably existing consent ONE TIME ONLY.</label>
           </div>
+          <div class="form-check">
+            <input
+              type="checkbox"
+              value="{{ settingsForm.get('cacheLot') }}"
+              formControlName="cacheLot"
+              id="cacheLot"
+            />
+            <label for="cacheLot">Use protocol cache</label>
```

Comment:
as we are in fintech ui, it would be more clear to write
Use protocol cache of TPP Server

Reply:
ok